src/html_support_files/odoc.css

```
@@ -246,24 +246,42 @@ body {
 }
 
 body {
-  margin-left: calc(10vw + 20ex);
-  margin-right: 4ex;
-  margin-top: 20px;
-  margin-bottom: 50px;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 4ex;
 }
 
 body.odoc {
-  max-width: 100ex;
+  max-width: 122ex;
```

If I can count, this makes the space for the content smaller: `100ex` before, vs `90ex` (`122ex - 28ex - 4ex`).

I liked the large content better!
```suggestion
  max-width: 132ex;
```

---

I liked it smaller, but thats fine